scripts/bert/finetune_classifier.py

```diff
@@ -69,7 +69,7 @@
 test_batch_size = args.test_batch_size
 lr = args.lr
 
-ctx = mx.cpu() if args.gpu is None or args.gpu == '' else mx.gpu()
+ctx = mx.cpu() if not args.gpu else mx.gpu()
 
 bert, vocabulary = bert_12_768_12(dataset_name='book_corpus_wiki_en_uncased',
```

Comment:
Would it be more flexiable to make `dataset_name` as an argument and `assert` when the `dataset_name` and `do_lower_case` doesn't align? e.g., dataset_name is 'uncased' but `do_lower_case=False` as below:

```
bert, vocabulary = bert_12_768_12(dataset_name='book_corpus_wiki_en_uncased',
                                  pretrained=True, ctx=ctx, use_pooler=True,
                                  use_decoder=False, use_classifier=False)
tokenizer = FullTokenizer(vocabulary, do_lower_case=False)
```

Reply:
Sounds good me.